### PR TITLE
Updated community packs page, added Tartarus

### PR DESCRIPTION
**Pull request by Roan-V**

### CODE REVIEW

File: docs/config/community-packs.rst [hunk 1]
```diff
@@ -2,14 +2,25 @@
...
+| To add your pack to this list, submit a PR to the `Docs Repository <https://github.com/PolyhedralDev/TerraDocs>`__
 
-This is expected as Terra v6 has not been released for long, and we have not built up sufficient
-documentation to provide developers with the means to develop new packs for it.
 
-If you wish to get more out of Terra, or tweak things to better fit your vision, we would recommend
-modifying the `Overworld`_ pack, which provides many preset configurations and variables to play
-around with. See the :doc:`/config/development/modifying-existing-pack` guide to get started.
+Overworld pack
+==========
```
Comment (Astrashh) [inline docs/config/community-packs.rst:10]: The header underline should contain the same amount of characters as the heading

Comment (Astrashh) [reply docs/config/community-packs.rst:10]: Sphinx warning for reference
```
/home/astrash/pro/terra/docs/docs/config/community-packs.rst:10: WARNING: Title underline too short.

Overworld pack
==========
```

File: docs/config/community-packs.rst [hunk 2]
```diff
@@ -2,14 +2,25 @@
...
+| **Authors:** Astrash, Sancires, Aureus
+| **Source:** https://github.com/PolyhedralDev/TerraOverworldConfig
+| **Terra Version:** 6.x
```
Comment (Astrashh) [inline docs/config/community-packs.rst:13]: Include newline after block

Sphinx warning:
```
/home/astrash/pro/terra/docs/docs/config/community-packs.rst:12: WARNING: Line block ends without a blank line.
```

File: docs/config/community-packs.rst [hunk 3]
```diff
@@ -2,14 +2,25 @@
...
+Overworld config pack for Terra which provides many preset configurations and variables to play
+around with!
+See the :doc:`/config/development/modifying-existing-pack` guide to get started.
 
-.. _Overworld: https://www.github.com/PolyhedralDev/TerraOverworldConfig
\ No newline at end of file
+Tartarus
+==========
```
Comment (Astrashh) [inline docs/config/community-packs.rst:19]: Same above heading

File: docs/config/community-packs.rst [hunk 4]
```diff
@@ -2,14 +2,25 @@
...
+| **Author:** Jaddot
+| **Source:** https://github.com/Jason-Ding19/Tartarus
+| **Terra Version:** 6.x
```
Comment (Astrashh) [inline docs/config/community-packs.rst:22]: Include newline same as above

**Review by Astrashh — COMMENTED** [review]
Looks good, sphinx gives some warnings which I've included in the review, see the Usage section of the readme to view these warnings locally

**Comment by Roan-V** [comment]
Apologies, neglected to look at the console logs.
Requested changes were applied.

**Pull request merged**